models/eprop_iaf_adapt.h

```diff
@@ -92,58 +97,69 @@ able to spike for an absolute refractory period :math:`t_\text{ref}`.
 An additional state variable and the corresponding differential equation
 represents a piecewise constant external current.
 
-Furthermore, the pseudo-derivative of the membrane voltage needed for e-prop
-plasticity is calculated:
-
-.. math::
-    \psi_j^t = \frac{\gamma}{v_\text{th}} \text{max}
-               \left(0, 1-\left| \frac{v_j^t-A_j^t}{v_\text{th}}\right| \right) \,.
-
-See the documentation on the ``iaf_psc_delta`` neuron model for more information
-on the integration of the subthreshold dynamics.
+See the documentation on the :doc:`iaf_psc_delta<../models/iaf_psc_delta/>` neuron model
+for more information on the integration of the subthreshold dynamics.
 
-The change of the synaptic weight is calculated from the gradient :math:`g` of
-the loss :math:`E` with respect to the synaptic weight :math:`W_{ji}`:
-:math:`\frac{\mathrm{d}{E}}{\mathrm{d}{W_{ij}}}=g`
+The change of the synaptic weight is calculated from the gradient :math:`g^t` of
+the loss :math:`E^t` with respect to the synaptic weight :math:`W_{ji}`:
+:math:`\frac{\text{d}{E^t}}{\text{d}{W_{ij}}}=g^t`
 which depends on the presynaptic
 spikes :math:`z_i^{t-1}`, the surrogate gradient / pseudo-derivative of the postsynaptic membrane
 voltage :math:`\psi_j^t` (which together form the eligibility trace
 :math:`e_{ji}^t`), and the learning signal :math:`L_j^t` emitted by the readout
 neurons.
 
+See the documentation on the :doc:`eprop_archiving_node<../models/eprop_archiving_node/>` for details on the surrogate
+gradients functions.
+
+In between two presynaptic spikes, the gradient is calculated for each time step until the cutoff time point, i.e., for
+:math:`t \, \epsilon \, \left[ t_\text{s}^{t-1}, \text{min} \left( t_\text{s}^{t-1} + {\Delta t}_\text{c},
+t_\text{s}^t \right) \right]`:
+
 .. math::
-  \frac{\mathrm{d}E}{\mathrm{d}W_{ji}} = g &= \sum_t L_j^t \bar{e}_{ji}^t, \\
-  e_{ji}^t &= \psi_j^t \left(\bar{z}_i^{t-1} - \beta \epsilon_{ji,a}^{t-1}\right)\,, \\
-  \epsilon^{t-1}_{ji,\text{a}} &= \psi_j^{t-1}\bar{z}_i^{t-2} + \left( \rho - \psi_j^{t-1} \beta \right)
-  \epsilon^{t-2}_{ji,a}\,. \\
+  \frac{\text{d}E^t}{\text{d}W_{ji}} &= g^t = L_j^t \bar{e}_{ji}^t \,, \\
+  e_{ji}^t &= \psi_j^t \left( \bar{z}_i^{t-1} - \beta \epsilon_{ji,a}^{t-1} \right) \,, \\
+  \epsilon^{t-1}_{ji,\text{a}} &= e_{ji}^t + \rho \epsilon_{ji,a}^{t-2} \,. \\
 
 The eligibility trace and the presynaptic spike trains are low-pass filtered
 with some exponential kernels:
```

Comment:
```suggestion
The eligibility trace and the presynaptic spike trains are subjected to low-pass
filtering using exponential kernels:
```
I feel the word "some" is somewhat vague.

Reply:
Fixed in https://github.com/JesusEV/nest-simulator/pull/16/commits/9163aedd43fd15810a7cf2151d23fa639b49dbe8